Gems/Atom/Feature/Common/Code/Source/DiffuseGlobalIllumination/DiffuseProbeGridFeatureProcessor.cpp

```diff
@@ -152,6 +152,11 @@ namespace AZ
         {
             AZ_PROFILE_SCOPE(AzRender, "DiffuseProbeGridFeatureProcessor: Simulate");
 
+            if (!RHI::RHISystemInterface::Get()->GetDevice()->GetFeatures().m_rayTracing)
```

Comment:
Please remove this check, the DiffuseProbeGridFeatureProcessor is active even on non-RT platforms to support baked DiffuseProbeGrids.